docs/configuring-playbook-bot-draupnir.md

```diff
@@ -146,7 +146,10 @@ matrix_bot_draupnir_configuration_extension_yaml: |
 Draupnir supports two methods to receive reports in the management room.
 
 The first method intercepts the report API endpoint of the client-server API, which requires integration with the reverse proxy in front of the homeserver.
-While this playbook uses reverse proxies, it does not yet implement this.
+If you are using traefik, this playbook can set this up for you:
+```yaml
+matrix_bot_draupnir_web_abuse_reporting_enabled: true
```

Comment:
Should the default way of enabling this feature be `matrix_bot_draupnir_web_abuse_reporting_enabled` or `matrix_bot_draupnir_abuse_reporting_enabled`?

The first option requires less variables and thus less complexity.

The second option seems more intuitive, since from a user perspective this has nothing to do with web, the abuse reports are just sent as messages to the moderation room.

Reply:
I see your point! I guess keeping it `matrix_bot_draupnir_abuse_reporting_enabled` makes sense too. Feel free to do it that way